frontend/src/metabase/admin/databases/components/DatabaseEditApp/Sidebar/Sidebar.unit.spec.tsx

```diff
@@ -22,7 +22,7 @@ import Sidebar from "./Sidebar";
 const NOT_SYNCED_DB_STATUSES: InitialSyncStatus[] = ["aborted", "incomplete"];
 
 function getModal() {
-  return document.querySelector(".Modal") as HTMLElement;
+  return document.querySelector("[data-element-id=modal]") as HTMLElement;
```

Comment:
another reason to not use CSS class in testing moving forward is that we don't transform CSS module when testing, so we have to rely on another static property instead. e.g. `data-testid` or `data-element-id` in this case.

We opt for `data-element-id` primarily when it's used in the code not just for testing, but there's no discussion whether we should use `data-element-id` or `data-testid` for test-only code.

What do you think?

Reply:
I think I would prefer using `data-testid` for consistency. I like the idea of having a catch-all tag that can be used for testing and for usage within the code but I think that we should really bias ourselves twoards using test IDs even if it makes the code a little more obtuse.